rest-api/tools/import_organizations.py

```diff
@@ -201,14 +216,28 @@ def _cleanup_old_entities(self, session, row_list, dry_run):
     if orgs_to_remove:
       org_id_list = []
       for org in orgs_to_remove:
-        log_prefix = '(dry run) ' if dry_run else ''
-        logging.info(log_prefix + 'Deleting old Organization no longer in Google sheet: %s', org)
         old_org = self.org_dao.get_by_external_id(org)
-        org_id_list.append(old_org.organizationId)
-        self.deletion_count += 1
+        if old_org and old_org.isObsolete != ObsoleteStatus.OBSOLETE:
+          org_id_list.append(old_org.organizationId)
+          self.deletion_count += 1
+        elif old_org.isObsolete == ObsoleteStatus.OBSOLETE:
```

Comment:
Same as above, check that old_org is not None first.

Reply:
done